Code/Framework/AzFramework/AzFramework/Network/IToolingConnection.h

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
```

Comment:
IToolingConnection / AzToolsFramework is a bit weird with the mixing of the gerund form. The noun used inside cmake is also simply 'Tools'. Did you consider IToolsConnection and associated?

Reply:
Sure, I can make that change.